.editorconfig

```diff
@@ -8,6 +8,6 @@ charset = utf-8
 trim_trailing_whitespace = true
 indent_style = tab
 
-[*.{md,yml,yaml,json,toml,htm,html,js,css,svg,sh,bash,fish}]
+[*.{mdx,yml,yaml,json,toml,htm,html,js,css,svg,sh,bash,fish}]
```

Comment:
```suggestion
[*.{md,mdx,yml,yaml,json,toml,htm,html,js,css,svg,sh,bash,fish}]
```

No?

Reply:
In this PR (#1432) we moved from .md to .mdx but the editor config was not updated
Can do another PR if needed

Reply:
Only the docs are mdx. There is still md elsewhere (readme, changelog etc). No reason not to list both here though. No worries about a separate PR.

Reply:
You're right! I'll add both :)